<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>967</width>
    <height>499</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="menu_layout">
     <item>
      <widget class="QLabel" name="title_label">
       <property name="text">
        <string>Titlu</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="title_box"/>
     </item>
     <item>
      <widget class="QLabel" name="name_label">
       <property name="text">
        <string>Autor</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="name_box"/>
     </item>
     <item>
      <widget class="QLabel" name="genre_label">
       <property name="text">
        <string>Gen</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="genre_combobox"/>
     </item>
     <item>
      <widget class="QPushButton" name="search_button">
       <property name="text">
        <string>Cauta</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
